### PR TITLE
RFC5084: add CCM parameters to EncryptionAlgorithm

### PR DESCRIPTION
**Pull request by aalba6675**
This to support parsing BouncyCastle FIPS BCFKS keystores which use AES-CCM.
https://github.com/bcgit/bc-java/blob/master/prov/src/main/java/org/bouncycastle/jcajce/provider/keystore/bcfks/BcFKSKeyStoreSpi.java

### CODE REVIEW

File: asn1crypto/algos.py [hunk 1]
```diff
@@ -454,6 +454,15 @@ class Pbes1Params(Sequence):
     ]
 
 
+class CcmParams(Sequence):
+    # https://tools.ietf.org/html/rfc5084
+    # aes_ICVlen: 4 | 6 | 8 | 10 | 12 | 14 | 16
+    _fields = [
+        ('aes_nonce', OctetString),
+        ('aes_ICVlen', Integer),
```
Comment (wbond) [inline asn1crypto/algos.py]: So far all of the field names in the library have been all lowercase (if I recall correctly). For consistency, can we make this lowercase also?

**Comment by wbond** [comment]
You wouldn't perchance have a file we could add as a test fixture to make sure we maintain this functionality moving forward?

**Comment by aalba6675** [comment]
1. Convert to lowecase for consistency
2. Add a test to parse CcmParams objects

**Comment by wbond** [comment]
Awesome, thanks!

**Pull request merged**